extensions/PhabBugz/Extension.pm

```diff
@@ -14,10 +14,30 @@ use warnings;
 use parent qw(Bugzilla::Extension);
 
 use Bugzilla::Constants;
-use Bugzilla::Extension::PhabBugz::Feed;
+use Bugzilla::Extension::PhabBugz::Constants;
 
 our $VERSION = '0.01';
 
+sub template_before_process {
+    my ( $self, $args ) = @_;
+    my $file = $args->{'file'};
+    my $vars = $args->{'vars'};
+
+    return
```

Comment:
I don't think we should support this on the old interface.
also perhaps two 'return unless ...' statements here would be better than one?